test/unit/resources/auditd_rules_test.rb

```diff
@@ -10,29 +10,37 @@
     resource = MockLoader.new(:centos7).load_resource('auditd_rules')
     _(resource.send('lines')).must_equal [
        '-a always,exit -F arch=b64 -S open,openat -F exit=-EACCES -F key=access',
+       '-a always,exit -F arch=b32 -S open,openat -F exit=-EPERM -F key=access',
        '-a always,exit -F arch=b32 -S chmod,fchmod,fchmodat -F auid>=500 f24!=0 -F key=perm_mod',
+       '-a always,exit -S all -F path=/usr/bin/chage -F perm=x -F auid>=1000 -F auid!=-1 -F key=privileged',
+       '-a always,exit -S all -F path=/usr/bin/mount -F perm=x -F auid>=1000 -F auid!=-1 -F key=privileged',
        '-w /etc/ssh/sshd_config -p rwxa -k CFG_sshd_config',
+       '-w /etc/sudoers -p wa',
+       '-w /etc/private-keys -p x',
     ]
   end
 
   it 'auditd_rules syscall interface' do
     resource = MockLoader.new(:centos7).load_resource('auditd_rules')
     _(resource.send('syscall', 'open').send('rules')).must_equal [
-      {:syscall=>"open", :list=>"exit", :action=>"always", :fields=>["arch=b64", "exit=-EACCES", "key=access"], :arch=>"b64", :exit=>"-EACCES", :key=>"access"}
```

Comment:
Unfortunately I had to update the test file of the auditd_rules resource since the new and old both use the same command `auditctl -l` and therefore the same mock command.